webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionHistoryDrawer.vue

```diff
@@ -0,0 +1,145 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2022 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <exo-drawer
+    @closed="closed"
+    ref="versionHistoryDrawer"
+    class="versionHistoryDrawer"
+    show-overlay
+    right>
+    <template slot="title">
+      {{ $t('versionHistory.label.title') }}
+    </template>
+    <template slot="content">
+      <div
+        v-if="isLoading"
+        class="text-center mt-5">
+        <v-progress-circular
+          indeterminate
+          color="primary" />
+      </div>
+      <div
+        v-if="!isLoading && versions.length === 0"
+        class="text-center mt-5">
+        <p class="grey--text darken-1">
+          {{ $t('versionHistory.label.empty') }}
+        </p>
+      </div>
+      <v-list
+        class="ma-3">
+        <v-list-item-group
+          active-class="bg-active">
+          <v-slide-y-transition group>
+            <v-list-item
+              v-for="version in versions"
+              :key="version.id"
+              :class="[version.current? 'current_version' : '']"
+              @click="openVersion($event, version)"
+              class="history-line pa-2 mb-2 border-color border-radius d-block">
+              <version-card
+                @version-update-description="updateVersionDescription"
+                @restore-version="restoreVersion"
+                :version="version"
+                :can-manage="canManage"
+                :disable-restore-version="disableRestoreVersion"
+                :enable-edit-description="enableEditDescription" />
+            </v-list-item>
+          </v-slide-y-transition>
+        </v-list-item-group>
+      </v-list>
+    </template>
+    <template slot="footer">
+      <div
+        v-if="showLoadMore"
+        class="d-flex mx-4">
+        <v-btn
+          @click="loadMore"
+          class="primary--text mx-auto"
+          text>
+          {{ $t('versionHistory.button.loadMore') }}
+        </v-btn>
+      </div>
+    </template>
+  </exo-drawer>
+</template>
+
+<script>
+
+export default {
+  props: {
+    versions: {
+      type: Array,
+      default: () => {
+        return [];
```

Comment:
It's better to add null here all time for default values, else, it will be dropped once the value is set for the component, and this will bring more things to do for JS GC (instantiate an object that will be dropped just after).